src/Cassandra/SchemaParser.cs

```diff
@@ -840,8 +840,7 @@ public override Task<AggregateMetadata> GetAggregateAsync(string keyspaceName, s
                             StateFunction = row.GetValue<string>("state_func"),
                             FinalFunction = row.GetValue<string>("final_func"),
                             InitialCondition = row.GetValue<string>("initcond"),
-                            Deterministic = row.GetColumn("deterministic") != null &&
-                                            row.GetValue<bool>("deterministic"),
+                            Deterministic = row.GetColumn("deterministic") != null && row.GetValue<bool?>("deterministic").GetValueOrDefault(false),
```

Comment:
Isn't this a server bug? Or is it working as intended to get null values here?

Reply:
Based on the current status of the ticket, it looks like a bug: https://datastax.jira.com/browse/DSP-24606
What do we want to do here?

Reply:
let's remove all production code changes from this PR and keep the test changes and make the failing HCD tests skip when it's HCD with a message that refers `DSP-24606` (no full link since it's a private JIRA project).

The changes are trivial but I don't really want to do a release for the C# driver when this is something that will be fixed in HCD (and it's highly unlikely someone will run into this before an HCD fix is in place anyway).